app/views/users/confirmations/new.html.erb

```diff
@@ -5,7 +5,7 @@
 
   <div class="field">
     <%= f.label :email %><br />
```

Comment:
aiight @HacksForHugs gotcha. let's keep it for now, and then remove when we refactor